examples/_tests_mnist_stages2/config1.yml

```diff
@@ -17,11 +17,11 @@ stages:
   transform_params:
     transform: A.Compose
     transforms:
-      - transform: catalsyt.TensorToImage
+      - transform: C.TensorToImage
       - transform: A.Normalize
         mean: [0.1307]
         std: [0.3081]
-      - transform: catalsyt.ToTensor
+      - transform: C.ToTensor
```

Comment:
but why?

we have never called our framework `C`. using `catalsyt.ToTensor` is much clearer

Reply:
@Scitator

Reply:
first of all - **C**ata**lyst** :)
but, `Catalyst.ToTensor` looks too long and complicated

Reply:
`from catalyst import utils`

it's lowercase :)

and here: `transform: catalsyt.ToTensor` it means to use  `ToTensor` from sort of package `catalyst.`

Reply:
`C.ToTensor` is unclear and more complicated

Reply:
why do you think so? why is it unclear and more complicated?

Reply:
because, you will never know, what is `C.`, where else it's used

`smp.Unet` means you use Unet from module `segmentation_models_pytorch` also known as `smp`

`A.RandomCrop` means you use RandomCrop from `albumentations` also known as `A`

what does `C.ToTensor` mean?

Reply:
To peacefully end the argue, I suggest adding functionality that allows you to add several different prefixes for the same method or class.

For example, `.add_from_module(t, prefix=["catalyst.", "C."])`.

Reply:
you know `albumentaions as A`, nevertheless from my perspective they are `albu` or `albumentations `... it's a matter of taste.

but the idea with  different prefixes looks good, you are right :)

Reply:
you can also add aliases for albumentations, like `albumentations`, `albu`, `A`

Reply:
albumentations uses `A.` in their first and examples :)

Reply:
* in their docs

Reply:
would you like to make changes during this PR, or is it okay for you to make them on https://github.com/catalyst-team/catalyst/pull/607 ?
GANs are waiting for you, @TezRomacH :)

Reply:
I am literally working on this right now

give me 30-40 minutes

Reply:
done